pyintelowl/cli/_utils.py

```diff
@@ -34,6 +34,7 @@ def get_status_text(status: str, as_text=True):
         "reported_without_fails": ("#73D216", str(Emoji("heavy_check_mark"))),
         "reported_with_fails": ("#CC0000", str(Emoji("warning"))),
         "failed": ("#CC0000", str(Emoji("cross_mark"))),
+        "killed": ("#CC0000", str(Emoji("cross_mark"))),
```

Comment:
We should choose different emojis I think. For killed and failed.

Reply:
Well actually I don't think there are any suitable emoji. So this can be left as is.